admin/server/server.go

```diff
@@ -34,7 +34,11 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-var minCliVersion = version.Must(version.NewVersion("0.28.0"))
+var _minCliVersion = version.Must(version.NewVersion("0.20.0"))
+var methodWiseMinCliVersion = map[string]*version.Version{
+	"UpdateProject":      version.Must(version.NewVersion("0.28.0")),
+	"UpdateOrganization": version.Must(version.NewVersion("0.28.0")),
+}
```

Comment:
1. Let's just use fully qualified API names (like `/rill.admin.v1.AdminService/UpdateProject`) here to avoid potential clashes in future
2. nit: call the var `_minCliVersionByMethod` to have consistent prefix